draft-irtf-cfrg-vdaf.md

```diff
@@ -354,16 +355,16 @@ In fact, our framework admits DAFs with slightly more
 functionality, computing aggregation functions of the form
 
 ~~~
-f(agg_param, meas_1, ..., meas_N) =
-    g(agg_param, meas_1) + ... + g(agg_param, meas_N)
+F(agg_param, meas_1, ..., meas_BATCH_SIZE) =
+    G(agg_param, meas_1) + ... + G(agg_param, meas_BATCH_SIZE)
```

Comment:
* capital letters for functions looks a little nicer in the *.txt file, which is what I'm reviewing.
* `N` is used in the overview section to denote the number of aggregators.